x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/bytes.test.tsx

```diff
@@ -85,7 +79,47 @@ describe('Processor: Bytes', () => {
     const processors = getProcessorValue(onUpdate, BYTES_TYPE);
     expect(processors[0].bytes).toEqual({
       field: 'field_1',
-      ...defaultBytesParameters,
+    });
+  });
+
+  test('saves with common fields set', async () => {
```

Comment:
Thanks for adding this, @cuff-links! I think for even better discoverability we could move this test to a separate file, something like 'common_fields.test.ts'. Something to make it explicit that this checks the fields used in every processor and it's not needed in individual processor tests. WDYT?

Reply:
Split the test out.